cmake/Modules/MaCh3Dependencies.cmake

```diff
@@ -2,34 +2,17 @@
 # download CPM.cmake
 file(
   DOWNLOAD
-  https://github.com/cpm-cmake/CPM.cmake/releases/download/v0.38.8/CPM.cmake
+  https://github.com/cpm-cmake/CPM.cmake/releases/download/v0.40.2/CPM.cmake
   ${CMAKE_CURRENT_BINARY_DIR}/cmake/CPM.cmake
 )
 include(${CMAKE_CURRENT_BINARY_DIR}/cmake/CPM.cmake)
 
-#Luke's handing cmake modules which Neutrino hep experiments might want
-CPMFindPackage(
-      NAME CMakeModules
-      GIT_TAG stable
-      GITHUB_REPOSITORY NuHepMC/CMakeModules
-      DOWNLOAD_ONLY
-  )
-include(${CMakeModules_SOURCE_DIR}/NuHepMCModules.cmake)
-include(NuHepMCUtils)
-
 # Check if CUDA was found
 if(MaCh3_GPU_ENABLED)
   include(${CMAKE_CURRENT_LIST_DIR}/CUDASetup.cmake)
 endif()
 
-include(ROOT)
-if(NOT TARGET ROOT::ROOT)
-  cmessage(FATAL_ERROR "MaCh3 Expected dependency target: ROOT::ROOT")
-endif()
-
-if(ROOT_VERSION VERSION_LESS 6.18.00)
-  cmessage(FATAL_ERROR "Using ROOT version smaller than 6.18.0, this may lead to unexpected results")
-endif()
+find_package(ROOT 6.18 REQUIRED)
```

Comment:
Can you also update this
https://github.com/mach3-software/MaCh3/blob/d6414032bbf683c940f2801d2662b52f4e50fb16/cmake/Templates/MaCh3Config.cmake.in#L23